util/intest/common.go

```diff
@@ -17,4 +17,4 @@
 package intest
 
 // InTest checks if the code is running in test.
-const InTest = false
+const InTest = true
```

Comment:
You don't need to modify it, it will be automatically set to true during testing.

Reply:
I think you can modify this comment.